code/drasil-example/gamephysics/lib/Drasil/GamePhysics/DesignDec/Vector.hs

```diff
@@ -154,7 +154,7 @@ vectRotate = funcDef "vectRotate" [vV1, vV2] (Vect Rational)
     )
   ]
   
-vectUnrotate = funcDef "vectUnrotate" [vV1, vV2] (Vect Rational)
+vectUnrotate = funcDef "vectUnrotate" [vV1, vV2] (Vect Real)
```

Comment:
Out of curiosity, were these changes automatically caught by the type-checker (or somewhere down the line), or did you extrapolate yourself?

Reply:
For this one, I extrapolated myself.